### ISSUE TITLE
[DATA] CVE-2019-18680 is missing version information

### ISSUE THREAD

**Issue by DerDakon** [nluedtke/linux_kernel_cves#209]
**Change Type Requested**
Update

**CVE id number**
CVE-2019-18680

**References**
Breaks: https://github.com/gregkh/linux/commit/c4e97b06cfdc5213494c22dd5c2b41ff8b15e0ee 4.4.179
Fixes: https://github.com/gregkh/linux/commit/91573ae4aed0a49660abdad4d42f2a0db995ee5e 4.4.195

**Comment by cschmid-xse** [comment]
Upstream "breaks" is cb66ddd156203daefb8d71158036b27b0e2caf63 in v5.1-rc4
Upstream "fixes" is not seen so far except in kernel 4.4.195

**Comment by DerDakon** [comment]
This was caused by a backport, the NULL check that was added in 4.4.195 is not in upstream and that free code was meanwhile refactored and the newly introduced function doing this already checks for NULL. So the bug never existed in mainline.

**Comment by nluedtke** [comment]
fixed with 11db6289df0a5eeb917b3cf558e73ff38acdae25

**Comment by DerDakon** [comment]
Not completely:

```JSON
"CVE-2019-18680": {
        "affected_versions": "unk to unk",
```

And it still shows up in e.g. 3.12 and 5.9 as "fix not seen in stream" even if these branches were never affected.
